modules/videopress/utility-functions.php

```diff
@@ -73,13 +73,16 @@ function videopress_get_video_details( $guid ) {
  *
  * Modified from https://wpscholar.com/blog/get-attachment-id-from-wp-image-url/
  *
+ * @deprecated since 8.4.0
  * @todo: Add some caching in here.
```

Comment:
We could remove it :)

Reply:
Ah, OK. Remove the entire function `videopress_get_attachment_id_by_url()` ?

Reply:
No, I mean remove this "@todo" comment, sorry

Reply:
Ah, good idea.

Reply:
Removed in https://github.com/Automattic/jetpack/commit/85b14a0d38bbdddbcb7a1ba88e968587546848b5#diff-66c3e0655a6e9d684e7898c2c70171c3L77